tests/test_edgeql_introspection.py

```diff
@@ -1590,3 +1590,18 @@ async def test_edgeql_schema_inheritance_computed_backlinks_01(self):
             } LIMIT 0''',
             []
         )
+
+    async def test_edgeql_introspection_secret(self):
+        await self.con.execute(
+            '''
+            CREATE EXTENSION pgcrypto;
+            CREATE EXTENSION auth;
+            '''
+        )
+        await self.assert_query_result(
```

Comment:
Could you move this test case into `test_http_ext_auth.py` or somewhere else `auth` is already created? Creating it is pretty slow.